specification/appconfiguration/AppConfiguration/routes.tsp

```diff
@@ -346,7 +354,7 @@ op putKeyValue is appConfigOperation<
     @body
     entity?: KeyValue;
   },
-  KeyValue &
+  Body<KeyValue> &
```

Comment:
Why is this necessary?

Reply:
https://github.com/Azure/azure-rest-api-specs/pull/29887#discussion_r1684975812